core/vmpu/src/armv7m/vmpu_armv7m.c

```diff
@@ -300,79 +300,97 @@ void vmpu_load_box(uint8_t box_id)
 
 extern int vmpu_region_bits(uint32_t size);
 
+/* Compute the MPU region size for the given BSS sections and stack sizes.
+ * The function also updates the region_start parameter to meet the alignment
+ * requirements of the MPU. */
+static uint32_t vmpu_acl_stack_region_size(uint32_t * const region_start, uint32_t const bss_size,
```

Comment:
Consider removing `const` from `uint32_t * const region_start`, to make explicit that `region_start` will be modified. As written, it requires a careful eye to understand where the `const` applies to, as it isn't in a form that coders expect and can sight-read easily. And `const` for value types isn't really that important to state explicitly, anyway. It's worth the readability improvement to remove 'const' here: be obvious and not subtle.